example/src/main/java/org/apache/rocketmq/example/quickstart/Consumer.java

```diff
@@ -48,6 +49,7 @@ public static void main(String[] args) throws InterruptedException, MQClientExce
          * }
          * </pre>
          */
+        consumer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
```

Comment:
May cause the official website quick start to set namesrv through the command line to fail.